### ISSUE TITLE
Issue with static push publishing when pushing with a default language other than English

### ISSUE THREAD

**Issue by swicken-dotcms** [dotCMS/core#22266]
**Describe the bug**
There appears to be issues with static push publishing when you attempt to push on an instance with a language that has a default other than English. 

Related ticket: https://dotcms.zendesk.com/agent/tickets/107070
Reproduced on: 22.03

**To Reproduce**
Start an environment in a non-US English default language

```
## LANGUAGE AND CODING INFO
CHARSET = text/html;charset=UTF-8
DEFAULT_LANGUAGE_COUNTRY = Germany
DEFAULT_LANGUAGE = de_DE
DEFAULT_LANGUAGE_CODE = de
DEFAULT_LANGUAGE_STR = German
DEFAULT_LANGUAGE_COUNTRY_CODE = DE
```
Once you have an environment in this default language, please contact me for additional context/reproduction. This has to do with pulling content of specific base types. 

We also have an existing hotfix for 22.01 to analyze/compare against I can provide.

**Comment by swicken-dotcms** [comment]
[hotfix-GIT-20653_v22.01 (1).zip](https://github.com/dotCMS/core/files/8781159/hotfix-GIT-20653_v22.01.1.zip)

Here is a copy of the hotfix for this issue

**Comment by freddyucv** [comment]
**Can't reproduce** it work for me

**Comment by bryanboza** [comment]
Fixed, tested on release-22.07 and now I'm able to push a single user without problems

**Comment by bryanboza** [comment]
Fixed, tested with the provided steps and it is working as expected. Tested on release 22.07 / Docker
